factcast-site/documentation/content/setup/properties.md

```diff
@@ -85,17 +85,23 @@ grpc.client.factstore.keep-alive-without-calls=true
 
 Further details can be found here : `net.devh.boot.grpc.client.config.GrpcChannelProperties`.
 
+#### FactCast client specific
+
+| Property        | Description           | Default  | Example |
+| ------------- |:-------------|:-----|:-----|
+|factcast.grpc.client.catchup-batchsize|Request a batchsize in catchup phase. Produces larger message and better compression. Remember that this setting increases the memory requirements, as well as the individual message size so depending on you Fact-payload size, and this setting, you may wan to increase the allowed max-in/out limits of GRPC (defaulting to ~4mb per message). Our tests have shown that values >100 seem to have an insignificant impact - your mileage may vary. Setting is valid since 0.3.9.|50|100
```

Comment:
```suggestion
|factcast.grpc.client.catchup-batchsize|Request a batchsize in catchup phase. Produces larger message and better compression. Remember that this setting increases the memory requirements, as well as the individual message size so depending on you Fact-payload size, and this setting, you may want to increase the allowed max-in/out limits of GRPC (defaulting to ~4mb per message). Our tests have shown that values >100 seem to have an insignificant impact - your mileage may vary. Setting is valid since 0.3.9.|50|100
```